src/main.cpp

```diff
@@ -110,6 +189,38 @@ int main(int argc, char* argv[])
 
       return 0;
    }
+   if (app_options.user_string_dict().length() > 0) {
+      std::string path =
+         std::regex_replace(app_options.user_string_dict(), std::regex("\\\\"), "\\");
+      path = std::regex_replace(path, std::regex("\""), "");
+      if (fs::exists(path))
+         read_dictionary(path.c_str());
+      else {
+         std::cout << "Error: file '"s << path << "' does not exist\n";
+
+         return 0;
+      }
+   }
+   else
+      initialize_internal_dictionary();
```

Comment:
Again a small style thing that's my fault. if statements (and as a result else statements) that don't fit on one line I typically surround with angle brackets.
```suggestion
   else {
      initialize_internal_dictionary();
   }
```